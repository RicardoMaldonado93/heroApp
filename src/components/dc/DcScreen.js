import React from 'react'
import { HeroesList } from '../ui/heroes/HeroesList'

export const DcScreen = () => {

    const publisher = "DC Comics";

    return (
        <div>
            <h1>DC SCREEN</h1>
            <hr />

            <HeroesList 
                publisher={ publisher }
            />
        </div>
    )
}
