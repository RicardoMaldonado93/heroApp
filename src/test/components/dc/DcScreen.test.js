import {shallow} from 'enzyme';
import React from 'react';
import { DcScreen } from '../../../components/dc/DcScreen';

describe('Unitary test for <DcScreen />', () => {
    
    const wrapper = shallow( <DcScreen /> );

    test('should be match with snapshot', () => {
        expect( wrapper ).toMatchSnapshot();     
    });
    
});
