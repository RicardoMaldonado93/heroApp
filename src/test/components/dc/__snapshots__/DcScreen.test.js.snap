// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unitary test for <DcScreen /> should be match with snapshot 1`] = `
<div>
  <h1>
    DC SCREEN
  </h1>
  <hr />
  <HeroesList
    publisher="DC Comics"
  />
</div>
`;
